chapter07/sxg/ex_7_5.pl

```diff
@@ -0,0 +1,20 @@
+#!perl
+use warnings;
+use strict;
+
+my $find=0;
+my $line;
+
+print "I found:\n";
+while (<>) {
+	$line = $_;
+
+  	if (/[A-Z][a-z]+/) {

```

Comment:
First letter upper case [A-Z],
then an infinite number of(+) small cases [a-z]

This is what i thought the requirement ;)